client/lib/application/src/application-lifecycle/components/Loading.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+import Icon from 'react-fa';
+
+const Loading = () => {
+    return (
+        <ThreeColumns
```

Comment:
So if you want to reuse the Loading component, you always get three flex columns along with it?